backend/src/main/kotlin/com/tngtech/apicenter/backend/connector/database/mapper/configurer/VersionMappingConfigurer.kt

```diff
@@ -17,6 +17,7 @@ class VersionMappingConfigurer constructor(private val versionConverter: Version
         orikaMapperFactory.classMap(VersionEntity::class.java, Version::class.java)
             .field("versionId.version", "version")
             .field("content", "content")
+            .field("language", "language")
```

Comment:
Do you need to specifiy that? I thought Orika can map identically called fields automatically.

Reply:
Yes, they are required. (When omited all file uploads fail, just checked.)